example/dj/apps/app/tests/fieldsets.py

```diff
@@ -0,0 +1,80 @@
+from unittest.case import TestCase
```

Comment:
upravoval jsem RFS tak jsme rovnou napsal testy toto je hodně důležité aby to fungovalo dobře